app/src/main/java/org/mozilla/fenix/tabstray/browser/BrowserTrayInteractor.kt

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray.browser
+
+import mozilla.components.concept.tabstray.Tab
+import mozilla.components.feature.tabs.TabsUseCases
+import org.mozilla.fenix.tabstray.TabsTrayInteractor
+
+/**
+ * For interacting with UI that extends from [BaseBrowserTrayList] and other browser tab tray views.
+ */
+interface BrowserTrayInteractor {
+
+    /**
+     * Select the tab.
+     */
+    fun onOpenTab(tab: Tab)
+
+    /**
+     * Close the tab.
+     */
+    fun onCloseTab(tab: Tab)
+
+    /**
+     * Enable or disable multi-select mode.
+     */
+    fun onMultiSelect(enabled: Boolean)
+}
+
+/**
+ * A default implementation of [BrowserTrayInteractor].
+ */
+class DefaultBrowserTrayInteractor(
+    private val trayInteractor: TabsTrayInteractor,
+    private val selectTabUseCase: TabsUseCases.SelectTabUseCase,
+    private val removeUseCases: TabsUseCases.RemoveTabUseCase
+) : BrowserTrayInteractor {
+
+    /**
+     * See [BrowserTrayInteractor.onOpenTab].
+     */
+    override fun onOpenTab(tab: Tab) {
+        selectTabUseCase.invoke(tab.id)
+        trayInteractor.navigateToBrowser()
+    }
+
+    /**
+     * See [BrowserTrayInteractor.onCloseTab].
+     */
+    override fun onCloseTab(tab: Tab) {
+        removeUseCases.invoke(tab.id)
+    }
+
+    /**
+     * See [BrowserTrayInteractor.onMultiSelect].
+     */
+    override fun onMultiSelect(enabled: Boolean) {
+        // TODO
```

Comment:
What will be done here? Could use some more detail in the to do.

Reply:
Sorry, I spoke offline with Codrut who will be working on this in https://github.com/mozilla-mobile/fenix/issues/18443.

This patch was blocking that work, so I just left a to-do comment to appease the linter, but it might end up being an unnecessary interactor method when it's worked on.

Reply:
Updated the comment to reference the issue number.